src/log/log_slot.c

```diff
@@ -531,7 +531,7 @@ __wti_log_slot_join(WT_SESSION_IMPL *session, uint64_t mysize, uint32_t flags, W
     uint64_t time_start, time_stop, usecs;
     int64_t flag_state, new_state, old_state, released;
     int32_t join_offset, new_join, wait_cnt;
-    bool closed, diag_yield, raced, slept, unbuffered, yielded;
+    bool closed, diag_yield, raced, slept, unbuffered, yielded, force_unbuffered;
```

Comment:
Please alphabetize the variables in this declaration. That's our coding convention. See: https://github.com/wiredtiger/wiredtiger/blob/develop/CONTRIBUTING.rst